crates/astria-conductor/src/config.rs

```diff
@@ -9,6 +9,13 @@ use serde::{
     Serialize,
 };
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub enum CommitLevel {
```

Comment:
Bikesheddy: Could add some utilities to his like:
```
fn is_soft_only(&self) -> bool {
    matches!(self, Self::SoftOnly)
```
and the same for the others.

The conditions in conductor would then look like:
```rust
if !commit_level.is_soft_only() {
```